### ISSUE TITLE
Sink: Decouple sync of offsets with the `put()` function

### ISSUE THREAD

**Issue by azapps** [azapps/kafka-backup#3]
Ideally this job would be scheduled independently of the delivery of new messages from Kafka Connect. Currently the offsets are synchronized every time, new records are pushed to Kafka Connect.

**Comment by itadventurer** [comment]
See https://github.com/getdreams/kafka-backup/pull/20

**Comment by nc-ruth** [comment]
@itadventurer  - could you please update this issue, is it still relevant?

**Comment by itadventurer** [comment]
Yes, this issue is still relevant but I currently do not have any time continuing the development of kafka-backup.

**Comment by nc-ruth** [comment]
@itadventurer - thats cool :)
Thank you for the update.
